modules.d/95nvmf/parse-nvmf-boot-connections.sh

```diff
@@ -297,6 +297,22 @@ parse_nvmf_discover() {
     return 0
 }
 
+nvmf_timeout_udev_rule() {
+    local timeout
+
+    timeout=$(getarg rd.timeout)
+    case $timeout in
+        0 | "")
+            timeout=-1
+            ;;
+    esac
+    mkdir -p /etc/udev/rules.d
+    printf -- 'ACTION=="add|change", SUBSYSTEM=="nvme", KERNEL=="nvme*", ATTR{ctrl_loss_tmo}="%d"\n' "$timeout" \
+        > /etc/udev/rules.d/90-nvmf-timeout.rules
```

Comment:
Just to clarify, this rule will only be present in the initramfs, not in the real rootfs, right?

Reply:
Yes, that's the idea. If we establish a connection from the real root, this connection is obviously not required to mount the real root. 

People may still want to use a different timeout than the default, but in the real root we have other means to configure that. We certainly should not use `rd.timeout` in the real root.

Reply:
Thanks for explaining. I was not sure where this part of the script is executed. (no dracut hacker here)